docs/usage.md

```diff
@@ -135,3 +135,52 @@ We recommend adding the following line to your environment to limit this (typica
 ```bash
 NXF_OPTS='-Xms1g -Xmx4g'
 ```
+
+## Input specifications
+
+The input data can be passed to nf-core/mag in two possible ways using the `--input` parameter.
+
+### Direct FASTQ input (short reads only)
+
+The easiest way is to specify directly the path (with wildcards) to your input FASTQ files. For example:
+
+```bash
+--input 'path/to/data/sample_*_R{1,2}.fastq.gz'
+```
+
+This input method only works with short read data and will assign all files to the same group. By default, this group information is only used to compute co-abundances for the binning step, but not for group-wise co-assembly (see the parameter docs for [`--coassemble_group`](https://nf-co.re/mag/parameters#coassemble_group) and [`--binning_map_mode`](https://nf-co.re/mag/parameters#binning_map_mode) for more information about how this group information can be used).
+
+Please note the following additional requirements:
+
+* Files names must be unique
```

Comment:
Really? I thought these do not need to be unique any more.

Reply:
only when using the "direct" FASTQ input using wildcards, because then `fromFilePairs` extracts the sample name from the base filenames.

Reply:
Right.